### ISSUE TITLE
Language undefined isn't supported

### ISSUE THREAD

**Issue by PoslinskiNet** [vadimdemedes/trevor#54]
Hello

Not sure what might be the case, but Yesterday it was working fine.

Today, I receiving the following error:

![screen shot 2017-02-24 at 10 31 08](https://cloud.githubusercontent.com/assets/1891508/23298230/20a25d3e-fa7d-11e6-8e9b-6d636cb4259f.png)

Docker: 1.13.1 (15353)
Node:  7.5.0 (npm v4.1.2)

Any idea what might be the case?

Would be nice to have some better debug/verbose output to simplify debug.

Thanks in advance for any help.

**Comment by SamVerschueren** [comment]
Can you share your `.travis.yml` file?

**Comment by PoslinskiNet** [comment]
```
sudo: required
services:
  - docker
node_js:
  - "4"
env:
  COMPOSE_VERSION: 1.9.0

cache:
  directories:
    - $HOME/.npm
    - $HOME/.cache # includes bowers cache

before_install:
  - curl -L https://github.com/docker/compose/releases/download/${COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
  - chmod +x docker-compose
  - sudo mv docker-compose /usr/local/bin
  - docker login -u "$DOCKER_USER" -p "$DOCKER_PASSWORD"

install:
  - docker-compose build

script:
  - docker-compose up

addons:
  hosts:
  - api.myapp.local
  - app.myapp.local
```

**Comment by vadimdemedes** [comment]
I thought `language` was a required field in `.travis.yml`. Will fix, thanks!

**Comment by vadimdemedes** [comment]
Fixed in `2.3.0`.
